### PR TITLE
Remove obsolete `--multiquery` parameter (follow-up to #63898), pt. III

### PR DESCRIPTION
**Pull request by rschu1ze**
This PR is follow-up to https://github.com/ClickHouse/ClickHouse/pull/63898, removing the obsolete `-n` parameters from tests to avoid confusion why the parameter is still there and future cargo-cult copy-pasting of obsolete stuff into new tests.

Also see https://github.com/ClickHouse/ClickHouse/pull/67435 and https://github.com/ClickHouse/ClickHouse/pull/67749

### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by robot-ch-test-poll2** [comment]
*This is an automated comment for commit 634222dbec4fe62a765e29c5423ae6e290c3fe83 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/67964/634222dbec4fe62a765e29c5423ae6e290c3fe83/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Docs check</td><td>Builds and tests the documentation</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/67908/0bb8d1a8ede4906a1cdc75af2dbf1e190ff355fc/docs_check.html">✅ success</a></td></tr>
<tr><td>Unit tests</td><td>Runs the unit tests for different release types</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/7af334bd468b3474d5cfb673b9ea5f15664e938b/unit_tests__release_.html">✅ success</a></td></tr>
<tbody>
</table>
</details>

File: tests/queries/0_stateless/02532_send_logs_level_test.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 # shellcheck source=../shell_config.sh
 . "$CURDIR"/../shell_config.sh
 
-$CLICKHOUSE_CLIENT -nm -q "
+$CLICKHOUSE_CLIENT -m -q "
```
Comment (evillique) [inline tests/queries/0_stateless/02532_send_logs_level_test.sh:11]: From the looks of it `-m` is also not needed

**Review by evillique — APPROVED** [review]

File: tests/queries/0_stateless/02532_send_logs_level_test.sh (hunk 1 above)
Comment (rschu1ze) [reply tests/queries/0_stateless/02532_send_logs_level_test.sh:11]: True but let's keep this PR simple and stupid. We can do other cleanups separately.

**Comment by rschu1ze** [comment]
Pushing again to resolve merge conflicts.

**Pull request merged**